Translations of `STDCM Help Section` component updated

weblate opened the pull request:
> [!IMPORTANT]
> Do not merge this PR: See instead https://github.com/OpenRailAssociation/osrd/pull/13009 

Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/STDCM Help Section](https://hosted.weblate.org/projects/osrd/stdcm-help-section/).

Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/stdcm-help-section/horizontal-auto.svg)

flomonster reviewed (approved):
LGTM

weblate committed:
front: {{ component_slug  }}: translated using weblate ({{ language_name }})

Currently translated at {{ stats.translated_percent }}% ({{ stats.translated }} of {{ stats.all }} strings)

Translation: {{ project_name }}/{{ component_name }}
Translate-URL: {{ url }}

Signed-off-by: {{ author }}

front: {{ component_slug  }}: translated using weblate ({{ language_name }})

Currently translated at {{ stats.translated_percent }}% ({{ stats.translated }} of {{ stats.all }} strings)

Translation: {{ project_name }}/{{ component_name }}
Translate-URL: {{ url }}

Signed-off-by: {{ author }}

Co-authored-by: Khoyo <khoyobegenn@gmail.com>
Co-authored-by: Manuela Silva <mmsrs@sky.com>
Signed-off-by: Khoyo <khoyobegenn@gmail.com>
Signed-off-by: Manuela Silva <mmsrs@sky.com>

emersion commented on front/public/locales/fr/stdcm-help-section.json:
```diff
@@ -53,6 +53,10 @@
               "value": "Vos requêtes sont pour le moment possibles uniquement entre Perrigny et Miramas, entre J-7 et J-1 17 heures, et ne prennent pas en compte les ATEs, matières dangereuses, et gabarits GB."
+            },
+            {
+              "type": "text",
+              "value": "<a href='mailto:{{stdcmFeedbackMail}}' target=\"_blank\">Contactez-nous</a>"
```
Sounds like this is an extraneous block?

flomonster replied:
Indeed, I suggest fixing this in a dedicated frontend PR.

clarani commented on front/public/locales/pt/stdcm-help-section.json:
```diff
@@ -1 +1,326 @@
+    "application": {
+      "content": [
+        {
+          "type": "texto",
```
Block types should remain in english, otherwise it can't be interpreted in our code
```suggestion
          "type": "text",
```

<img width="517" height="953" alt="image" src="https://github.com/user-attachments/assets/85c82710-2401-42bf-bd97-0c30cbcbfef2" />

flomonster replied:
Fixed
<img width="504" height="1064" alt="image" src="https://github.com/user-attachments/assets/06d1220d-ff38-4e7d-9aa9-68fe12c73544" />

clarani commented on front/public/locales/pt/stdcm-help-section.json:
```diff
@@ -1 +1,326 @@
+                  "value": "velocidade máxima em relação à composição: é possível inserir um valor inferior ao pré-preenchido utilizando o código da composição"
+                }
+              ],
+              "type": "lista"
```
```suggestion
              "type": "list"
```

braintrue committed:
front: stdcm-help-section: add translation using weblate (Korean)

Signed-off-by: Changseop Yeom <ckdtjqdlgh@gmail.com>

braintrue committed:
front: stdcm-help-section: translated using weblate (Korean)

Currently translated at 1.5% (2 of 131 strings)

Translation: OSRD/STDCM Help Section
Translate-URL: https://hosted.weblate.org/projects/osrd/stdcm-help-section/ko/

Signed-off-by: Changseop Yeom <ckdtjqdlgh@gmail.com>

betubs committed:
front: stdcm-help-section: translated using weblate (German)

Currently translated at 3.0% (4 of 131 strings)

Translation: OSRD/STDCM Help Section
Translate-URL: https://hosted.weblate.org/projects/osrd/stdcm-help-section/de/

Signed-off-by: beTUBS <b.ehrenholz@tu-bs.de>

braintrue committed:
front: stdcm-help-section: translated using weblate (Korean)

Currently translated at 2.2% (3 of 131 strings)

Translation: OSRD/STDCM Help Section
Translate-URL: https://hosted.weblate.org/projects/osrd/stdcm-help-section/ko/

Signed-off-by: Changseop Yeom <ckdtjqdlgh@gmail.com>

clarani reviewed (approved):
LGTM :white_check_mark:
